src/config.ts

```diff
@@ -37,7 +37,7 @@ export async function getConfig(flags: Flags) {
   // `meow` is set up to pass boolean flags as `undefined` if not passed.
   // copy the struct, and delete properties that are `undefined` so the merge
   // doesn't blast away config level settings.
-  const strippedFlags = Object.assign({}, flags);
+  const strippedFlags = {...flags};
```

Comment:
For future me, this was really helpful to understand why this rule exists:
https://thecodebarbarian.com/object-assign-vs-object-spread.html

I am 50/50 on it, because semantically these two lines of code are doing ... well, the exact same thing because we were using the leading `{}` as the first parameter to create an immutable copy (neither do deep copies at the sub-object layer).